src/questionnaires/questionnaires.service.ts

```diff
@@ -81,6 +81,8 @@ export class QuestionnairesService {
       .collection('answers');
     if ('userId' in filters) {
       query = query.where('userId', '==', filters.userId);
+    } else if ('sessionId' in filters) {
```

Comment:
Kjør 'if' i stedet for 'else if', vi ønsker å kunne filtrere på både 'userId' og 'sessionId' samtidig

Reply:
vil ikke det overskrive query?

Reply:
Firestore-spørringer kan ha flere where-statements, så tror det bare vil 'appende'. Gjerne sjekk d ut